how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define pipeline parameters\n",
+    "pipeline_parameters = {\n",
+    "    \"dayOfMonth\": \"@dayOfMonth('2022-01-01 08:00:00')\",\n",
+    "    \"dayOfWeek\": \"@dayOfWeek('2022-01-01 08:00:00')\",\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Create a schedule with pipeline parameter expressions for published pipeline endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_endpoint_schedule = Schedule.create_for_pipeline_endpoint(ws,\n",
+    "                                    name=\"MyPipelineEndpointSchedule\",\n",
+    "                                     pipeline_endpoint_id=pipeline_endpoint_id,\n",
+    "                                     experiment_name=\"Schedule-endpoint-pipeline-parameter-sample\",\n",
+    "                                     description=\"Create for pipeline endpoint with pipeline parameters\",      \n",
+    "                                     pipeline_parameters=pipeline_parameters,\n",
+    "                                     recurrence=recurrence\n",
+    "                                     ) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Update a schedule with pipeline parameter expressions for published pipeline  \n",
+    "    **Note**: if users want to update pipeline without parameters, please republish the new pipeline to the pipeline endpoint, otherwise users need to update schedule to update pipeline with parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "update_pipeline_parameters = {\n",
+    "    \"dayOfMonth\": \"@dayOfMonth(utcNow())\",\n",
+    "    \"dayOfWeek\": \"@dayOfWeek(utcNow())\",\n",
+    "}\n",
+    "\n",
+    "pipeline_endpoint_schedule.update(name=\"MyUpdatedPipelineEndpointSchedule\",\n",
+    "                description=\"Based on time, pipeline paramdter expression\",\n",
+    "                pipeline_parameters=update_pipeline_parameters,\n",
+    "                recurrence=update_recurrence)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Disable the schedule "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_endpoint_schedule.disable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Questions and Answers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Q**: Why are my input pipeline parameter expressions not resolved?  \n",
+    "**A**: There are two ways that invalid expressions can be treated.  \n",
```

Comment:
The answer is organized from dev perspective. Please rewrite from customer view.